packages/story-editor/src/components/canvas/karma/backgroundCopyPaste.karma.js

```diff
@@ -234,13 +239,7 @@ describe('Background Copy Paste integration', () => {
     await fixture.events.click(fixture.editor.canvas.pageActions.addPage);
   }
   async function gotoPage(index /* 1-indexed */) {
-    const carousel = getElementByQueryAndMatcher(
-      'div[role="listbox"]',
-      getByAttribute('aria-label', /Pages List/i)
-    );
-    const pageAtIndex = carousel.querySelectorAll('button[role="option"]')[
-      index - 1
-    ];
+    const pageAtIndex = fixture.editor.footer.carousel.pages[index - 1].node;
```

Comment:
This wasn't actually required, but I thought the error had to do with this at first, so updated it anyway.